compiler-rt/lib/builtins/aarch64/sme-libc-mem-routines.S

```diff
@@ -252,7 +252,15 @@ DEFINE_COMPILERRT_FUNCTION_ALIAS(__arm_sc_memmove, __arm_sc_memcpy)
 #define zva_val  x5
 
 DEFINE_COMPILERRT_OUTLINE_FUNCTION_UNMANGLED(__arm_sc_memset)
-        dup     v0.16B, valw
+#  ifdef __ARM_FEATURE_SVE
```

Comment:
nit: why the space between the `#` and the` ifdef/else/endif` ?

Reply:
There was no real reason for the spaces, I'll remove them before merging.